pkg/okteto/namespace.go

```diff
@@ -168,3 +168,23 @@ func (c *namespaceClient) SleepNamespace(ctx context.Context, namespace string)
 
 	return nil
 }
+
+// DestroyAll deletes a namespace
+func (c *namespaceClient) DestroyAll(ctx context.Context, namespace string) error {
+	var mutation struct {
+		Space struct {
+			Id graphql.String
+		} `graphql:"destroyAllInSpace(id: $id, includeVolumes: $includeVolumes)"`
+	}
+	// includingVolumes so everything is cleaned up by default with this cmd
+	variables := map[string]interface{}{
+		"id":             graphql.String(namespace),
+		"includeVolumes": graphql.Boolean(true),
```

Comment:
Why always true? `Destroy` operation already receives a boolean to indicate if the volumes has to be destroyed or not. I think we should use the value of that flag. At the end, the user already has an option to delete volumes or not when deleting all from the UI

cc @pchico83 @rlamana

Reply:
i thought that "all" meant all... so making this default to true will also remove the volumes 🤷🏻‍♀️  
WDYT? no problem in changing it

Reply:
agree with @ifbyol. `all` means all the repos and services. we already have `-v` to destroy volumes

Reply:
added this value at the mutation